model-optimizer/extensions/middle/SliceConverter.py

```diff
@@ -36,80 +63,59 @@ class ConvertSlice(MiddleReplacementPattern):
     """
 
     enabled = True
-    op = "Slice"
     force_clean_up = True
+    op = "Slice"
 
     def run_after(self):
         from extensions.middle.pass_separator import MiddleStart
         return [MiddleStart]
```

Comment:
Please delete, as this code is useless -- `MiddleReplacementPattern` already has exactly the same method